app/controllers/pages_controller.rb

```diff
@@ -2,6 +2,7 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: [:home]
 
   def home
+    redirect_to alerts_path if user_signed_in? && current_user.role == "worker"
```

Comment:
We want both the resident and the worker to redirect to alerts_path if they are signed in, right?
Because depending on their role they will each get the partial alerts show view meant for their role.